### DISCUSSION TITLE
What happened when open many tabs with Payload?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [payloadcms/payload#5350] [Q&A] [answered]
Hi, I have a collection `Product`, it's fine if I open in one tab to create/update Product. But when I tried to open many tabs of product, each tab is different from others. I tried to save them quickly, but look like it load forever. What happened? Is this feature [`Rate Limiting Requests`] (https://payloadcms.com/docs/production/preventing-abuse#rate-limiting-requests)  ?  How to turn it off on develope mode or extend the rate limit ?

![image](https://github.com/payloadcms/payload/assets/15695252/ec757092-0061-4c13-8034-515ff11a026e)

**Comment by IRediTOTO** [comment]
I just tried to use local database use pgadmin4, it still loading forever, so I think this is PayloadCMS feature. But how to by pass or extend it ?

**Comment by IRediTOTO** [comment]
I tried this config but it looks like it does not work. 
```
export default buildConfig({   
    rateLimit: {
        skip: ({}) => true,
        max: 50000,
    },
  // ... 
})
```

**Comment by trananhmanh89** [comment, marked as answer]
Hi,
This issue occurs because webpack HMR (Hot Module Replacement) keeps the connection always live. The browser imposes limits on the number of connections that can be kept alive on the server. However, it's important to note that this issue only occurs during development and should not affect production.

**Reply by IRediTOTO** [reply]
Thank you for your info. VN here too :D
